api/registrations/serializers.py

```diff
@@ -393,17 +392,10 @@ def get_files_count(self, obj):
     def anonymize_registered_meta(self, obj):
         """
         Looks at every question on every page of the schema, for any titles
-        matching ANONYMIZED_TITLES.  If present, deletes that question's response
+        that have a contributor-input block type.  If present, deletes that question's response
         from meta_values.
         """
-        meta_values = obj.registered_meta.values()[0]
-        if is_anonymized(self.context['request']):
-            registration_schema = RegistrationSchema.objects.get(_id=obj.registered_schema_id)
-            for page in registration_schema.schema['pages']:
-                for question in page['questions']:
-                    if question['title'] in ANONYMIZED_TITLES and meta_values.get(question.get('qid')):
-                        del meta_values[question['qid']]
-        return meta_values
+        return self.anonymize_fields(obj, obj.registered_meta.values()[0])
```

Comment:
Consolidating anonymizing fields logic, because I can use this same logic to override the contributors questions before I register.  Also, not using ANONYMIZED_TITLES, even for old workflow, now.